deep_qa/data/instances/sequence_tagging/verb_semantics_instance.py

```diff
@@ -0,0 +1,173 @@
+from typing import Dict, List, Tuple
+
+import numpy
+from overrides import overrides
+
+from ..instance import TextInstance, IndexedInstance
+from ...data_indexer import DataIndexer
+
+
+class VerbSemanticsInstance(TextInstance):
+    """
+    A VerbSemanticsInstance is a :class:`TextInstance` that is a single sentence of text,
+    where that sentence has some associated categorical label.
+    """
+    def __init__(self, sentence: List[str], verb: (int, int), entity: (int, int), state_change: str, arg1: (int, int), arg2: (int, int), index: int=None):
```

Comment:
Hmm, I didn't think that `(int, int)` was a valid type declaration, but pylint doesn't appear to be complaining about it.  That's good to know.